### PR TITLE
CV2-2879: Populate Check API's `seeds.rb` file

### PR DESCRIPTION
**Pull request by vasconsaurus**
It’s easier for developer onboarding when there is an easy way to populate the database with some fake data. So we are implementing a seeds file that creates:

- one Team
- one User (admin)
- one Project
- Medias:  Claim, Link, Audio, Image, Video
- Imported Fact-Check
- Tipline Requests  
- Relationships between medias: Claim, Image, Audio

For medias and tipline requests we are creating medias with varying degrees of completeness:
- medias that have both claim_description and fact_check
- medias that have only claim_description
- medias that don't have either

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -1,7 +1,230 @@
...
-#
-# Examples:
-#
-#   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
-#   Mayor.create(name: 'Emanuel', city: cities.first)
+include SampleData
+require "faker"
+
+Rails.cache.clear
+
```
Comment (caiosba) [inline db/seeds.rb:5]: Sorry that we don't have a Linter in place... but can you please be sure that you're using two spaces for indentation?

File: db/seeds.rb [hunk 2]
```diff
@@ -1,7 +1,230 @@
...
+end
+
+def create_description(project_media)
+    Media.last.type == "Link" ? humanize_link(Media.find(project_media.media_id).url) : Faker::Company.catch_phrase
+end
+
+def add_claim_descriptions_and_fact_checks(user,n_project_medias = 6, n_claim_descriptions = 3)
+    ProjectMedia.last(n_project_medias).each { |project_media| ClaimDescription.create!(description: create_description(project_media), context: Faker::Lorem.sentence, user: user, project_media: project_media) }
+    ClaimDescription.last(n_claim_descriptions).each { |claim_description| FactCheck.create!(summary: Faker::Company.catch_phrase, title: Faker::Company.name, user: user, claim_description: claim_description) }
+end
```
Comment (caiosba) [inline db/seeds.rb:70]: Please move the method definitions to the beginning.

File: db/seeds.rb [hunk 3]
```diff
@@ -1,7 +1,230 @@
...
+    }
+end
+
+def create_blank(project, team)
+    ProjectMedia.create!(project: project, team: team, media: Blank.create!)
+end
+
+def create_claim_description(user, project, team)
+    ClaimDescription.create!(description: Faker::Company.catch_phrase, context: Faker::Lorem.sentence, user: user, project_media: create_blank(project, team))
+end
```
Comment (caiosba) [inline db/seeds.rb:114]: Please move method definitions to the beginning.

File: db/seeds.rb [hunk 4]
```diff
@@ -1,7 +1,230 @@
...
+    a.save!
+end
+
+add_claim_descriptions_and_fact_checks(user)
+
+
+# ##########
+p "Created — user: #{data[:user_name]} — email: #{user.email} — password : #{data[:user_password]}"
+
+
```
Comment (caiosba) [inline db/seeds.rb:230]: Please delete these two blank lines.

File: db/seeds.rb (hunk 1 above)
Comment (vasconsaurus) [reply db/seeds.rb:5]: for some reason my vscode was setup to indent four spaces 🙃 fixed it!

**Review by caiosba — APPROVED** [review]

**Pull request merged**